Kernel/Storage/RamdiskDevice.h

```diff
@@ -35,7 +35,8 @@ class RamdiskDevice final : public StorageDevice {
 
     Mutex m_lock { "RamdiskDevice" };
 
-    NonnullOwnPtr<Memory::Region> m_region;
+    PhysicalAddress m_start_address;
+    size_t m_length;
```

Comment:
Please explicitly initialize primitives.

Reply:
My bad here. @boricj I don't mind rebasing my branch and fixing it if you want to.